app.go

```diff
@@ -438,3 +449,81 @@ func executeCancelSell(cmd command) bool {
 
 	return true
 }
+
+func executeSetBuyAmount(cmd command) bool{
+	account := accountStore.GetAccount(cmd.UserID)
```

Comment:
Multiple references to cmd.UserID. Pull it into a local variable so you're not looking up object references.